README.md

```diff
@@ -219,8 +219,8 @@ end
 `app/components/modal_component.html.erb`:
 ```erb
 <div class="modal">
-  <div class="header"><%= header %></div>
-  <div class="body"><%= body %>"></div>
+  <div class="header"><%= @header %></div>
+  <div class="body"><%= @body %>"></div>
```

Comment:
`ModalComponent` is declaring `with_content_areas :header, :body` which should work without changing this template to use instance variables.

That's working for this component template in the test suite: https://github.com/github/actionview-component/blob/17069837bcadf2f6e122627f0fc2f90c73622cba/test/app/components/content_areas_component.html.erb